### PR TITLE
Updated hover frame sizing and color

### LINKED ISSUE #6378: Redesign: Element hover/droptarget frames
## Feature Description

Update element hover and droptarget frames based on [Figma](https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Post-Stable?node-id=5646%3A2).

- [x] Update droptarget frames for background, media, and shapes (note how it resembles the selection state for the background) - and make sure they also perform correctly for general hover
- [x] Update text box frame used for hover

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by barklund**
## Context

This updates the frame sizing, color and style for all elements including background.

See todo in #6378.

This also removes the hint for the blue drop targets, as this is now covered by the help center.

Finally, this removes the thick blue semitransparent border used when dropping media into shapes or other media.

## User-facing changes

This updates the hover styling, which is also used for "available drop target" highlighting:

| Type | Before | After |
|-|-|-|
| Hover background | Screen Shot 2021-02-15 at 00 28 14 | Screen Shot 2021-02-15 at 00 26 40 |
| Hover shape | Screen Shot 2021-02-15 at 00 27 59 | Screen Shot 2021-02-15 at 00 26 44 |
| Hover text | Screen Shot 2021-02-15 at 00 28 03 | Screen Shot 2021-02-15 at 00 26 48 |
| Hover media | Screen Shot 2021-02-15 at 00 28 09 | Screen Shot 2021-02-15 at 00 26 51 |
| Show drop targets | Screen Shot 2021-02-15 at 00 28 26 | Screen Shot 2021-02-15 at 00 27 33 |

### QA

This PR can be tested by following these steps:

1. Add a shape, text and media element to page
2. Hover each of the elements and the white background element and observe the hover outline - it's thin, light blue and fades in and out with .5s transition
3. Start draggin an image from the library
4. Observe that the droppable element (background, shape and media) shows the same outline as on hover) and that text does not.

### UAT

- [x] UAT should use the same steps as above.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6378

### CODE REVIEW

File: assets/src/edit-story/masks/frame.js
Comment (barklund) [inline assets/src/edit-story/masks/frame.js:84]: We should make sure to align this between this PR and #6250 once either is merged. Maybe move to some shared CSS helper and/or constants.

File: assets/src/edit-story/components/canvas/framesLayer.js
Comment (barklund) [inline assets/src/edit-story/components/canvas/framesLayer.js]: This is pretty magic - but it's added the offset of the background outline (as to not partially cover it). It should probably use a constant as well coming from that shared helper.

File: assets/src/edit-story/masks/frame.js
Comment (miina) [reply assets/src/edit-story/masks/frame.js:84]: @barklund Is there Figma for it as well? Sam told me that the drop target colors won't change, want to confirm if this has been reconsidered meanwhile.

Comment (barklund) [reply assets/src/edit-story/masks/frame.js:84]: Checking in [this thread](https://xwp.slack.com/archives/C01ESBB8ZN1/p1613421825239100).

Comment (barklund) [reply assets/src/edit-story/masks/frame.js:84]: Confirmed - and we remove the "Drop targets are blue" message.

File: assets/src/edit-story/masks/frame.js
Comment (miina) [inline assets/src/edit-story/masks/frame.js]:
```suggestion
  // Show an outline if hovering when not dragging
```

**Review by miina — APPROVED** [review]
LGTM pending removal of the thick drop-targeting border.

File: assets/src/edit-story/masks/frame.js
Comment (miina) [inline assets/src/edit-story/masks/frame.js:83]: These would be nice to have as a constant, too -- 4 being the gap and 8 being 2 times the gap.

File: assets/src/edit-story/masks/frame.js
Comment (miina) [inline assets/src/edit-story/masks/frame.js:148]: The blue border is still shown in the Helper btw -- probably that should get updated, too? (Not in this PR though!)

The drop-targeting feels a bit "rough" and unexpected to me now so would be good if Sam could also test it, however, this can be done in UAT.

**Review by miina — APPROVED** [review]
Still LGTM!

**Pull request merged**
